src/matrices.jl

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    UnitMatrix{T}(d)
+    UnitColumn{T}(d)
+    F{T}(d)
+    G{T}(N)
+    H{T}(d)
+
+Elementary matrices as defined by Aitchison 1986.
+
+## Examples
+
+```julia
+julia> UnitMatrix(5)
+julia> UnitColumn{Int}(2)
+julia> G(4)
+julia> H{Float64}(2)
+```
+
+When the type of the elements isn't explicitly defined, `Float64` is used.
+"""
```

Comment:
Notice that docstrings apply to a specific struct or function. Please split this string into multiple docstrings right before the structs defining the matrices. Also remove the leading empty line between the string and the struct so that Julia doc system can find the association.